<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>722</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>722</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="keMain1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>31</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOctoVendingMachine">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  bold 15pt &quot;Terminal&quot;;</string>
   </property>
   <property name="text">
    <string>OCTO VENDING MACHINE KELOMPOK 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>90</width>
     <height>133</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Graphics/TehPucuk-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>130</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Graphics/Nutriboost-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>95</width>
     <height>127</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>Teh Pucuk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="quantity_5">
      <property name="styleSheet">
       <string notr="true">font: 75 bold 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="harga_5">
      <property name="styleSheet">
       <string notr="true">font: bold 8pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>5000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBeli_5">
      <property name="text">
       <string>Beli</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>95</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>Nutriboost</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="quantity_6">
      <property name="styleSheet">
       <string notr="true">font: 75 bold 8pt &quot;System&quot;;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="harga_6">
      <property name="styleSheet">
       <string notr="true">font: bold 8pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>8000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBeli_6">
      <property name="text">
       <string>Beli</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tableWidget_2</zorder>
  <zorder>tableWidget</zorder>
  <zorder>keMain1</zorder>
  <zorder>labelOctoVendingMachine</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>verticalLayoutWidget_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
